### PR TITLE
[General] Text Files Needing Updates

### PR DESCRIPTION
**Pull request by zero-24**
>Some files in the repo need updates (issue is here more as a reminder than anything)
>
>- `.github/CONTRIBUTING.md` - Branch data
>- `libraries/src/Crypt/README.md` - Should reflect the added Sodium cipher
>
>May be others, but those two pop off the top of my head (OK, one's a nudge from George, but still).

cc @mbabker @wilsonge please review the files. Thanks.

### CODE REVIEW

File: libraries/src/Crypt/README.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
 # Important Security Information
 
-If you're going to use JCrypt in any of your plugins, make *sure* you use **JCryptCipherCrypto**; it's the only one that's cryptographically secure. (It's [version 1.1 of Defuse Security's encryption library](https://github.com/defuse/php-encryption)).
+If you're going to use JCrypt in any of your plugins, make *sure* you use **CryptoCipher** or **SodiumCipher**; They are only two that are cryptographically secure.
```
Comment (brianteeman) [inline libraries/src/Crypt/README.md]: Correct English is "Are the only"

But isn't this an upstream file?

Comment (mbabker) [reply libraries/src/Crypt/README.md]: `libraries/src` is all the CMS' namespaced libraries files (all came from `libraries/(cms|joomla|legacy)`.  `libraries/vendor` is all the upstream stuff.

Comment (brianteeman) [reply libraries/src/Crypt/README.md]: That's what happens when I comment from my phone :)

The comment about the English is still correct though

Comment (zero-24) [reply libraries/src/Crypt/README.md]: already fixed with https://github.com/joomla/joomla-cms/pull/17535/commits/14a33d9dcec892e2a24e37c95467f27d97ba1743 thanks!

File: libraries/src/Crypt/README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
 # Important Security Information
 
-If you're going to use JCrypt in any of your plugins, make *sure* you use **JCryptCipherCrypto**; it's the only one that's cryptographically secure. (It's [version 1.1 of Defuse Security's encryption library](https://github.com/defuse/php-encryption)).
+If you're going to use JCrypt in any of your extensions, make *sure* you use **CryptoCipher** or **SodiumCipher**; Are the only two that are cryptographically secure.
```
Comment (richard67) [inline libraries/src/Crypt/README.md]: Correct English would be: "**These** are the only two which are cryptographically secure".

Comment (brianteeman) [reply libraries/src/Crypt/README.md]: That's correct sorry I didn't see the full stop

Comment (zero-24) [reply libraries/src/Crypt/README.md]: fixed by the last commit thanks.

**Pull request merged**